packages/widgetbook/lib/src/addons/inspector_addon/inspector_addon.dart

```diff
@@ -34,7 +34,10 @@ class InspectorAddon extends WidgetbookAddon<bool> {
     return setting
```

Comment:
I don't think the conditional statement make sense after passing the `isEnabled` parameter.
We can remove both ternary operator and the scaffold as follows:

```suggestion
    return Inspector(
      isEnabled: setting,
      child: child,
    );
```

Reply:
That's fair enough, I assumed there was a reason that originally the addon was wrapping the `Inspector` in a `Scaffold`.

Reply:
Would also fix that the addon currently changes the background color which probably isn't desired behaviour.

Reply:
It probably just skipped the review during Hacktoberfest, but no reason of having it there.

Reply:
Ah I'll revert that change then 😁